### DISCUSSION TITLE
reencoding in h265, HEVC

### DISCUSSION THREAD

**Discussion by esquilo444** [datarhei/restreamer#681] [Q&A]
Good morning,
I would like to know if it was possible for the server to re-encode in h265?
I installed the cuda version on my server, no problem decoding HEVC with the p600 card installed on my server. But I would like to have the possibility of re-encoding the stream in hevc in hardware.
Is it possible ?
If so, is this a software or hardware limitation in my case?
thank you in advance for your insight...
Good day to you .

Jerome

**Comment by ioppermann** [comment]
This is currently a limitation in the UI. It is already on our feature request list.

**Reply by esquilo444** [reply]
Thanks a lot ...
You are Always the best ...😋
